hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/ChunkKeyHandler.java

```diff
@@ -128,17 +132,31 @@ protected void execute(OzoneClient client, OzoneAddress address)
         chunkDetails.setChunkName(chunkInfo.getChunkName());
         chunkDetails.setChunkOffset(chunkInfo.getOffset());
         chunkDetailsList.add(chunkDetails);
-        chunkPaths.add(getChunkLocationPath(containerData.getContainerPath())
-              + File.separator
-              + chunkInfo.getChunkName());
+        switch (chunkLayOutVersion) {
+        case FILE_PER_CHUNK:
+          chunkPaths.add(getChunkLocationPath(containerData
+                  .getContainerPath())
+                  + File.separator
+                  + chunkInfo.getChunkName());
+          break;
+        case FILE_PER_BLOCK:
+          chunkPaths.add(getChunkLocationPath(containerData
+                  .getContainerPath())
+                  + File.separator
+                  + keyLocation.getLocalID() + ".block");
+          break;
+        default:
+          throw new StorageContainerException("chunk strategy does not exist",
+                  ContainerProtos.Result.UNABLE_TO_FIND_CHUNK);
+        }
```

Comment:
I think we should move the code to give the chunk file path given the containerID/ContainerData into some common utilities class and use it across client/server so that any further changes won't break it..